tests/test_convert.py

```diff
@@ -545,6 +548,22 @@ def export_squeeze(filename, input_shape):
     return export(x, filename)
 
 
+def run_split(input):
+    a = tf.placeholder(tf.float32, shape=input.shape, name="input")
+    x1, x2 = tf.split(a, num_or_size_splits=2, axis=1)
+    y = tf.add(x1, x2)
```

Comment:
Can we remove this line?  We don't want to test Pond.add in this one, only split, but currently a bug in Pond.add will force this test case to fail.  There is a similar problem with `test_flatten` that i'm fixing in an upcoming PR, but I'd like to merge this one before I submit that.

Reply:
Let me remove this.